src/docfx/build/moniker/MonikersProvider.cs

```diff
@@ -20,17 +21,25 @@ public MonikersProvider(Config config, MonikerRangeParser monikerRangeParser)
             _rules.Reverse();
         }
 
-        public List<string> GetMonikers(Document file)
+        public List<string> GetMonikers(Document file, JObject yamlHeader = null)
```

Comment:
Add `MonikerRange` to `FileMetadata` and use it as the input

Reply:
where do we remove it? or it will  be output to .raw.page.json？

---
In reply to: [232901220](https://github.com/dotnet/docfx/pull/3610#discussion_r232901220) [](ancestors = 232901220)